pipeline/reformat_adobe_output.py

```diff
@@ -0,0 +1,47 @@
+import argparse
```

Comment:
Just a simple CLI for running locally. Can refactor and add s3 option if needed but didn't bother for now as it's super quick and can be ran locally for now.

Reply:
👍🏼 I imagine we'll look at how we separate out the Adobe API call, `data_to_document` step and this with @joel-wright at a later date.